### PR TITLE
Made calibrate chart sections collapsable

### PR DESCRIPTION
**Pull request by jryu01**
# Description

Screenshot 2024-09-30 at 6 40 42 PM
Screenshot 2024-09-30 at 6 40 51 PM

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue [hunk 1]
```diff
@@ -188,100 +188,106 @@ t d
...
+									<vega-chart
+										expandable
+										:are-embed-actions-visible="true"
+										:visualization-spec="preparedCharts[setting.selectedVariables[0]]"
+									/>
+								</template>
+							</AccordionTab>
+							<AccordionTab header="Errors">
+								<vega-chart
+									v-if="errorData.length > 0 && selectedErrorVariableSettings.length > 0"
```
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue]: I think it's more clean to hide the Errors heading when it's empty. I fixed it

File: packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue [hunk 2]
```diff
@@ -188,100 +188,106 @@ t d
...
+						</div>
+					</AccordionTab>
+				</Accordion>
+				<div v-if="!showSpinner">
+					<section ref="outputPanel" v-if="modelConfig && csvAsset">
+						<Accordion multiple :active-index="[0, 1, 2]">
+							<AccordionTab header="Parameters">
+								<template v-for="setting of selectedParameterSettings" :key="setting.id">
+									<vega-chart
+										v-if="preparedDistributionCharts[setting.selectedVariables[0]]"
```
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue:228]: Done

File: packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue [hunk 3]
```diff
@@ -188,100 +188,106 @@ t d
...
+					:summary-id="node.state.summaryId"
+					class="px-2"
+				/>
+				<Accordion :active-index="0">
+					<AccordionTab header="Loss">
+						<!-- Loss chart -->
+						<div ref="lossChartContainer">
 							<vega-chart
 								expandable
+								v-if="lossValues.length > 0 || showSpinner"
```
[the remark this answers is not in the document]
Comment (jryu01) [reply packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue]: Yeah, I changed it

**Review by YohannParis — APPROVED** [review]

**Review by mwdchang — APPROVED** [review]
Didn't test - code looks okay!

**Pull request merged**